dashboard-visualiser-jsreport/swarm.sh

```diff
@@ -1,71 +1,117 @@
 #!/bin/bash
 
-readonly ACTION=$1
-readonly MODE=$2
+declare ACTION=""
+declare MODE=""
+declare COMPOSE_FILE_PATH=""
+declare ROOT_PATH=""
+declare service_name=""
 
-COMPOSE_FILE_PATH=$(
-  cd "$(dirname "${BASH_SOURCE[0]}")" || exit
-  pwd -P
-)
+function init_vars() {
+  ACTION=$1
+  MODE=$2
 
-ROOT_PATH="${COMPOSE_FILE_PATH}/.."
-. "${ROOT_PATH}/utils/config-utils.sh"
-. "${ROOT_PATH}/utils/docker-utils.sh"
-. "${ROOT_PATH}/utils/log.sh"
+  COMPOSE_FILE_PATH=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")" || exit
+    pwd -P
+  )
+
+  ROOT_PATH="${COMPOSE_FILE_PATH}/.."
+
+  service_name="dashboard-visualiser-jsreport"
+
+  readonly ACTION
+  readonly MODE
+  readonly COMPOSE_FILE_PATH
+  readonly ROOT_PATH
+  readonly service_name
+}
+
+# shellcheck disable=SC1091
+function import_sources() {
+  source "${ROOT_PATH}/utils/docker-utils.sh"
```

Comment:
Let's declare a "UTILS_PATH" var

Reply:
cool done in [fad605b](https://github.com/jembi/platform/pull/191/commits/fad605bda9f8793cfe2803c151bd3e1577a6399f)